docs/src/main/sphinx/functions/list-by-topic.rst

```diff
@@ -313,10 +313,13 @@ For more details, see :doc:`json`
 * :func:`is_json_scalar`
 * :func:`json_array_contains`
 * :func:`json_array_length`
+* :ref:`json_exists() <json_exists>`
```

Comment:
do we want to have the () here?

Reply:
I think we want the `()`, because all the entries here render with `()`.